03week/ticTacToe.js

```diff
@@ -1,5 +1,46 @@
 'use strict';
```

Comment:
Make sure for future projects just to add the current working project. I shouldn't see pigLatin in here.

Reply:
I fixed the pull request so that the pig latin file is no longer showing up.  Assignment is ready to be graded.  Thank you!